### ISSUE TITLE
Add support for Git Bash on Windows

### ISSUE THREAD

**Issue by vivere-dally** [basefarm/aws-session-tool#39]
In the `_prereq` function, there is an [OSTYPE check](https://github.com/basefarm/aws-session-tool/blob/master/session-tool.sh#L190) based on the `openssl` location. A case for Git Bash can be added such that it will work on Windows:
```bash
	case $OSTYPE in
		darwin*	) _OPENSSL="/usr/bin/openssl";;
		msys* ) _OPENSSL="/mingw64/bin/openssl";;
		linux* | cygwin* ) _OPENSSL="openssl";;
		*) [[ $- =~ i ]] && echo >&2 "ERROR: Unknown ostype: $OSTYPE" ;;
	esac
```

**Comment by bjornrog** [comment]
Thank you for your report and suggested fix. It has been included in the branch PDA-639: 
https://github.com/basefarm/aws-session-tool/tree/PDA-639

This branch will be the basis for the next release.

**Comment by bjornrog** [comment]
We have done a lot of testing for session tool on GIt for Windows. We have not been able to get the save/restore function of session tool to work, due to issues with prompting the user for an encryption password.

The save/restore function has been disabled on Git for Windows. But otherwise it should now work.

The fix is included in version 1.6.4.
